contrib/julia-config.jl

```diff
@@ -23,6 +23,8 @@ function libDir()
     end
 end
 
+private_libDir() = joinpath(libDir(), "julia")
```

Comment:
this isn't necessarily accurate if this was set to something different (like in distro packaging layouts)